client/src/pages/SignupPage.jsx

```diff
@@ -0,0 +1,118 @@
+import { useState, useEffect } from 'react';
+import loginModel from "/src/assets/loginmodel.png";
+
+const SignupPage = () => {
+  const [width, setWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth);
+    };
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+  return (
+    <div className="h-screen bg-gradient-to-b from-color4 to-gray-400 flex justify-center">
+      <div className="h-full w-4/5 flex gap-10 justify-center items-center rounded-xl">
+        <div className="p-8 xl:w-1/3 sm:w-2/3 w-full">
+          <h1 className="text-white font-poppins text-5xl md:text-6xl font-bold mb-2 pb-8 text-center">
+            Sign up
+          </h1>
+          <form>
```

Comment:
Add a `Signup` component. As there will be a good amount of code for that, It will be easier for future changes. For now just add the whole form tag

This will address: #9